src/TimeTracker.Api/Properties/launchSettings.json

```diff
@@ -20,7 +20,7 @@
     "TimeTracker.Api": {
       "commandName": "Project",
       "launchBrowser": true,
-      "launchUrl": "/",
```

Comment:
was this the source of the double "/" lol

Reply:
i thought it was! but looks like maybe not